operations/client.go

```
@@ -152,3 +156,37 @@ func getJWTFromKanopy() cli.Command {
 		},
 	}
 }
+
+func setJWTInConfig() cli.Command {
+	return cli.Command{
+		Name:    "set-jwt",
+		Usage:   "generate a JWT token through kanopy-oidc and set it as \"jwt\" in the evergreen config file",
+		Aliases: []string{"sjwt"},
+		Action: func(c *cli.Context) error {
+			jwt, _ := runKanopyOIDCLogin()
```

Is it intentional to ignore the error here? After reviewing this function, it seems like it gives good error messages that would be useful to show to users.